index.html

```diff
@@ -165,7 +165,51 @@ <h1 class="crete w400 s72 lh88 ls037 orange">About me</h1>
                 </li>
             </ul>
         </section>
-        
+        <section id="contact">
+            <p class="inter nBlue2 s32 w700 lh44">I'm always<br>interested in hearing<br>about new projects,<br>so if you'd 
+               like to<br>chat please get in touch.
+            </p>
+            <form action="https://formspree.io/f/xlevqbpl" method="post">
+                <ul>
+                    <li class="formName">
+                        <label for="name" class="inter w400 s15 lh24">Full name</label>
+                        <input type="text" id="name" name="user_name" maxlength="30" required />
+                    </li>
+                    <li class="formEmail">
+                        <label for="mail">Email address</label>
+                        <input type="email" id="mail" name="user_email" pattern="^([a-zA-Z0-9_\-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([a-zA-Z0-9\-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$" maxlength="30" required/>
+                    </li>
+                    <li class="formMsg">                        
+                        <textarea id="message" name="user_message" placeholder="Enter text here" cols="30" rows="10" maxlength="500" required></textarea>
+                    </li>
```

Comment:
-  [ ] :thinking: I think it would be better to add a background to the text area to make it look like the the figma desgn:

  - Your design:
![yourTEXT](https://user-images.githubusercontent.com/97834160/205970624-f04000c5-7271-4bb7-b30e-ade33a1abdf2.png)


  - Figma design:
![figText](https://user-images.githubusercontent.com/97834160/205970650-b90ba458-5fe8-4a4c-9d29-53bdfbfdb791.png)